app/views/users/show.html.erb

```diff
@@ -14,6 +14,15 @@
   </section>
 
 
+  <section class="friends">
```

Comment:
I like this section- - maybe we can move this to the lefthand side for better style?